main.go

```diff
@@ -38,5 +39,5 @@ func run(_ *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	return s.ListenAndServe(ctx, config.HTTPSListenPort, config.HTTPListenPort, nil)
+	return s.ListenAndServe(ctx, config.HTTPSListenPort, config.HTTPListenPort, &server.ListenOpts{DisplayServerLogs: debugconfig.Debug})
```

Comment:
Those logs should pretty much always be set. I shouldn't have to run with `--debug` to see panics imo. They are error logs.

Reply:
Then we should drop that `DisplayServerLogs` option and always turn on "logging" (remember that only error-level output comes out of this process)

Reply:
No, we still need the option to keep the old behavior for other projects (k3s, rke2). Steve, Rancher and webhook will turn it on by default (eg: `DisplayServerLogs: true`) but the other projects won't (or they might, but we don't decide this).